samples/plugins/postgres_jdbc/connectionResolver.tdr

```diff
@@ -17,5 +17,8 @@
         </attribute-list>
       </required-attributes>
     </connection-normalizer>
+    <connection-properties>
+        <script file='connectionProperties.js'/>
```

Comment:
can you clean-up the whitespace in this file too.

Reply:
Fixed, thanks!